mimic/rest/maas_api.py

```diff
@@ -345,7 +330,7 @@ def list_entities(self, request, tenant_id):
         metadata['next_marker'] = None
         metadata['next_href'] = None
         request.setResponseCode(200)
-        return json.dumps({'metadata': metadata, 'values': entities})
+        return json.dumps({'metadata': metadata, 'values': entities}).encode('utf-8')
```

Comment:
Yeah, I wasn't sure if encoding the response on the way out was necessary or not. Let me try taking all those calls out and see if it still works.